docs/ops/generation/EyeLike_9.md

```diff
@@ -0,0 +1,142 @@
+## EyeLike <a name="EyeLike"></a> {#openvino_docs_ops_generation_EyeLike_9}
+
+**Versioned name**: *EyeLike-9*
+
+**Category**: *Generation*
+
+**Short description**: *EyeLike* operation generates shift matrix.
```

Comment:
I think we need to mention that it generates identity matrix or ones on the diagonal